### PR TITLE
Initial app skeleton

### PR DESCRIPTION
**Pull request by Kirosoft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+- [Usage](#usage)
+- [Getting Started](#getting-started)
+- [Migration](#migration)
+- [Unit tests](#unit-tests)
+- [Acceptance tests](#acceptance-tests)
+
+## Usage
+```
+get     /health                     # test endpoint for the application
+get     /scribe/<id_>                 # get a deed with an id in the URL
```
Comment (danielmarker) [inline README.md]: This will need updating to say sign a need rather than get

Comment (Kirosoft) [inline README.md]: yes, will need a total update. This is just the skeleton to build then jenkins pipeline. I will delete this for now.

**Comment by Phemus** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 # dm-scribe-api
+
+The Scribe API is a JSON API which provides signing services
```
Comment (endamccormack) [inline README.md]: Could use a bit more in the readme, you have an endpoint you haven't documented

File: application/esec/views.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from flask import Blueprint
+
+esec_bp = Blueprint('esec', __name__,
+                    template_folder='templates',
+                    static_folder='static')
+
+
+@esec_bp.route('/<esec_reference>', methods=['GET'])
```
Comment (endamccormack) [inline application/esec/views.py]: all these microservices will eventually be on the same endpoint so you probably should add a more descriptive endpoint like ```/sign/<esec_reference>``` or ```/create-user/<esec_reference>``` depending on what its doing

Comment (Kirosoft) [inline application/esec/views.py]: yes, there is no code here yet. This is just to test the CI pipeline at the moment

**Comment by endamccormack** [comment]
:+1: Good to go

**Pull request merged**